### ISSUE TITLE
apk webview only

### ISSUE THREAD

**Issue by eurohost** [android-js/androidjs#144]
Hi,

Would it be possible to pack only the html or even better a url in the apk without nodejs support

Also the next step after every apk build is to sign it, could it be possible to put the playstore.jks file in a folder  file and sign the apk with it. 

apksigner sign --ks ./key.jks --ks-pass pass:PASS --key-pass pass:PASS --out ./signed.apk  ./unsigned.apk

Thanks

**Comment by Chhekur** [comment]
@eurohost  yup we are working on a separate `SDK` where you can actually build static app without `node` using `Android JS`

**Comment by Chhekur** [comment]
@eurohost we have implemented that webview only SDK, you just need to get the latest version of `AndroidJS Builder` and you'll good to go.
```sh
$ npm install -g androidjs-builder@2.2.2
```
